opentechinstitute/commotion-router
# Rate limit ssh attempts to WAN zone

Incorporates firewall changes suggested by @raniarho in #116 with some modifications.
Partially addresses #30.

To test:
1. Flash the node and configure it as a gateway
2. Log in and make note of the public-facing IP address.
3. Connect another machine to the same network as the node, but not to the node itself. You should not receive an IP address from the node or be inside the commotion network.
4. SSH to the node and enter an invalid password until you are disconnected (3 attempts), then immediately repeat. On the fourth attempt within 1 minute your connection should be denied.
